src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -152,8 +152,7 @@ public static void Compose(object[] importers, Type[] typesToExport, params Expo
                 batch.AddExport(item);
             }
 
-            using var container = new CompositionContainer(catalog);
```

Comment:
The `using` meant the container was disposed as soon as the method ending, which could then throw exceptions when the imported objects are disposed.